src/hash.h

```diff
@@ -191,6 +191,39 @@ class CHashVerifier : public CHashWriter
     }
 };
 
+/** A writer stream that computes a SHA256 hash. */
+class SHA256Writer
+{
+private:
+    CSHA256 ctx;
+
+    const int nType;
```

Comment:
18c4d69f8d771aa596a321cf1d6c8e6e106d42a9

nit, should fix format.

Reply:
done